conf/application.conf

```diff
@@ -113,6 +113,7 @@ auditing {
 mongodb {
   uri = "mongodb://localhost:27017/sole-trader-identification"
   timeToLiveSeconds = 14400
```

Comment:
This should probably default to false then we can override in environment config when we need to use it